kong/clustering/services/sync/rpc.lua

```diff
@@ -177,6 +177,57 @@ local function is_rpc_ready()
 end
 
 
+local function lmdb_upsert(db, t, delta, opts, is_full_sync)
+  local delta_type = delta.type
+  local delta_entity = delta.entity
+
+  -- upsert the entity
+  -- delete if exists
+  local old_entity, err = db[delta_type]:select(delta_entity)
+  if err then
+    return false, err
```

Comment:
We should return `nil, err`, not a boolean.

Reply:
done